support/remote_health_monitoring/using-insights-operator.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="using-insights-operator"]
+= Using Insights Operator
+include::_attributes/common-attributes.adoc[]
+ifdef::openshift-dedicated[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+endif::[]
+:context: using-insights-operator
+
+toc::[]
+
+The Insights Operator periodically gathers configuration and component failure status and, by default, reports that data every two hours to Red Hat. This information enables Red Hat to assess configuration and deeper failure data than is reported through Telemetry. Users of {product-title} can display the report in the Insights Advisor service on link:https://console.redhat.com/openshift[Red Hat Hybrid Cloud Console].
```

Comment:
The same applies to this line, you could consider replacing "link:[https://console.redhat.com/openshift[Red Hat Hybrid Cloud Console]" with "{cluster-manager-url}" for consistency with this docs library.

Reply:
see above